src/components/organisms/TableHeaderGroup/styles.module.scss

```diff
@@ -0,0 +1,29 @@
+@import 'styles/variables';
+@import 'styles/mixins';
+
+//HeaderItem
+.headingWrapper {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: start;
+  align-items: center;
+  grid-gap: 10px;
+
+  input {
+    order: -1;
+  }
+
+  .sortingButton {
+    width: max-content;
+    svg g path,
+    svg path {
+      fill: $color-input-main;
+    }
+    svg {
+      padding: 0;
+    }
+    &:focus {
+      box-shadow: inset 0 0 0 2px transparent;
```

Comment:
**[blocker]**

Discussed this in the office, but you should contact Martine and discuss this part again. Would prefer not to break Accessbility for something that might look a bit better visually. Every component that the user can interact with should have some visible focused style